src/locales/en.ts

```diff
@@ -591,7 +591,7 @@ export default {
     pairing: {
       title: 'Hotspot Pairing Required',
       subtitle:
-        'Press the button on your Helium Hotspot, or power-cycle your RAK Hotspot Miner.',
+        'Press the button or power-cycle your Helium or 3rd party Hotspot.',
```

Comment:
this implies that you need to power cycle the helium hotspot.

Recommended change ->
"Place your Hotspot in Bluetooth pairing mode."

Reply:
agree with this suggested change

Reply:
@shawaj do you want to update this? Or want us to do it?